examples/dpi/dpi.go

```diff
@@ -0,0 +1,374 @@
+package main
+
+import (
+	"bytes"
+	"encoding/binary"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"hash/fnv"
+	"io/ioutil"
+	"os"
+	"regexp"
+	"sync/atomic"
+	"time"
+
+	"github.com/flier/gohs/hyperscan"
```

Comment:
Is everything good with dependencies here? Too small changes for new dependence? Maybe like gopacket?

Reply:
gohs requires pre-install some not-go dependencies
So I just added setup-deps.sh script, and use it CI on Ubuntu 16.04
And then make -C example/dpi